### PR TITLE
[except.throw,except.handle] Move lvalue specification for copies

### LINKED ISSUE #5091: [except.throw] doesn't specify the type of the lvalue denoting the exception object
[except.throw]/3:
> Throwing an exception copy-initializes a temporary object, called the _exception object_.
> An lvalue denoting the temporary is used to initialize the variable declared in the matching handler  ([except.handle]).

Is there any wording defining the type of this lvalue? Also, https://timsong-cpp.github.io/cppwp/except.handle#14 is worded in terms of initialization from object, not expression. Does it need a separate issue?

### PR DESCRIPTION
**Pull request by jensmaurer**
Fixes #5091

### CODE REVIEW

**Comment by jensmaurer** [comment]
@languagelawyer , this should address your concerns.

**Comment by languagelawyer** [comment]
Still doesn't say anything about the lvalue type.

**Comment by jensmaurer** [comment]
Do you want "an lvalue of the type of the exception object, designating the exception object"?

File: source/exceptions.tex [hunk 1]
```diff
@@ -655,11 +652,12 @@
 of type \tcode{E}, as follows:
 \begin{itemize}
 \item
-if \tcode{T} is a base class of \tcode{E}, the variable is
-copy-initialized\iref{dcl.init} from the corresponding base class subobject
+if \tcode{T} is a base class of \tcode{E},
+the variable is copy-initialized\iref{dcl.init}
+from an lvalue designating the corresponding base class subobject
```
Comment (languagelawyer) [inline source/exceptions.tex:657]:
```suggestion
from an lvalue of type \tcode{T} designating the corresponding base class subobject
```

File: source/exceptions.tex [hunk 2]
```diff
@@ -655,11 +652,12 @@
...
 of the exception object;
 \item otherwise, the variable is copy-initialized\iref{dcl.init}
-from the exception object.
+from an lvalue designating the exception object.
```
Comment (languagelawyer) [inline source/exceptions.tex:660]:
```suggestion
from an lvalue of type \tcode{E} designating the exception object.
```

**Review by languagelawyer — CHANGES_REQUESTED** [review]

**Comment by jensmaurer** [comment]
Sounds good.

**Pull request merged**